.kokoro/docker/docs/requirements.txt

```diff
@@ -8,9 +8,9 @@ argcomplete==3.5.3 \
     --hash=sha256:2ab2c4a215c59fd6caaff41a869480a23e8f6a5f910b266c1808037f4e375b61 \
```

Comment:
Ignore the changes in this file. The source of truth lives in another repository. We're planing remove this file from Github (Googlers see b/390135527)